### PR TITLE
Fix scripted test because of sbt downgrade

### PR DESCRIPTION
**Pull request by mkurz**

### CODE REVIEW

**Comment by mkurz** [comment]
@jroper Here we go

**Comment by jroper** [comment]
Could you either downgrade `project/build.properties` to `1.2.8`, or update the `validate` task in `build.sbt` to do a `^scripted`?

**Comment by jroper** [comment]
(I think this PR is going to fail if you don't do one of the above).

File: src/sbt-test/sbt-web-build-base/check-settings/test [hunk 1]
```diff
@@ -1,7 +1,10 @@
 > ^update
 
 # Make sure resolution was done for both sbt 0.13 and sbt 1.0
-$ exists target/scala-2.10/sbt-0.13/update/update_cache_2.10/output
-$ exists target/scala-2.12/sbt-1.0/update/update_cache_2.12/output
+$ exists target/scala-2.10/sbt-0.13/resolution-cache/com.typesafe.sbt/check-settings/scala_2.10/sbt_0.13/0.1-SNAPSHOT/resolved.xml.xml
+$ exists target/scala-2.12/sbt-1.0/resolution-cache/com.typesafe.sbt/check-settings/scala_2.12/sbt_1.0/0.1-SNAPSHOT/resolved.xml.xml
+# When switching to coursier:
+#$ exists target/scala-2.10/sbt-0.13/update/update_cache_2.10/output
+#$ exists target/scala-2.12/sbt-1.0/update/update_cache_2.12/output
```
Comment (mkurz) [inline src/sbt-test/sbt-web-build-base/check-settings/test]: I kept it as a comment for future updates.

**Comment by mkurz** [comment]
Green.

**Comment by jroper** [comment]
1.2.3 is released.

**Pull request merged**